src/components/GroupButton/index.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { RemoveCircle, AddCircleOutline } from "@material-ui/icons";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+
+class GroupedButtons extends React.Component {
+  state = { counter: 0 };
+
+  handleIncrement = () => {
+    this.setState((state) => ({ counter: state.counter + 1 }));
+  };
+
+  handleDecrement = () => {
+    this.setState((state) => ({ counter: state.counter - 1 }));
+  };
+  render() {
+    const displayCounter = this.state.counter > 0;
+
+    return (
+      <div>
+        <TextField
+          id="guest"
+          disabled
+          label="Guest"
+          InputLabelProps={{
+            shrink: true,
+          }}
+          variant="outlined"
+          style={{ position: "relative" ,
+          
+        }}
+        ></TextField>
+        <RemoveCircle
+          onClick={this.handleDecrement}
+          style={{
+            position: "absolute",
+            left: "28.5rem",
```

Comment:
You have not worked on this... You only need to make this left margin to be **`25.5rem`**. this will make it at the start of input as the increment button is at the end.
And also make sure you worked on the responsiveness because in small size screens buttons are scattering away. Even the content is sticking into the footer.